tailwind.config.js

```diff
@@ -17,6 +17,55 @@ module.exports = {
         lg: "1400px",
         xl: "1800px",
         "hide-remote-console": { raw: "(max-width: 600px), (max-height: 550px)" },
+        // custom media queries that also use the min-height property
```

Comment:
As Sergey mentioned in [this comment](https://elninotech.slack.com/archives/C01HGHENHRS/p1677141216468529?thread_ts=1677095177.610109&cid=C01HGHENHRS) I think we should not rely a lot on these media height breakpoints since the size of the box can be bigger on a device with a smaller height. But I still think it's a good idea to add them since it still can be used for some things.

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/119003472/220921889-16b3d152-3903-4065-bf46-05ae26073966.png">
<img width="1431" alt="image" src="https://user-images.githubusercontent.com/119003472/220921996-cddcecd6-22d2-4b3b-95a4-c8bd629bf823.png">

Reply:
Yeah, I think we should combine these approaches, for some components it might be enough to have only Tailwinf breakpoints, but for same we have to use box height